content/docs/setup/kubernetes/multicluster-install/index.md

```diff
@@ -267,3 +280,53 @@ $ kubectl delete -f $HOME/istio-remote.yaml
 {{< text bash >}}
 $ helm delete --purge istio-remote
 {{< /text >}}
+
+## Stable access to Istio control plane services
+In the above procedure, endpoint IPs of Istio services are gathered and used to
+invoke Helm that will create Istio services on the remote clusters. This updates
+kube-dns in the remote clusters so it can be used to resolve the Istio service names.
+Since Kubernetes pods don't have stable IPs, restart of any Istio service pod in
+the control plane cluster will cause its endpoint to be changed. Therefore, any
+connection made from remote clusters to that endpoint will be broken. This is 
+documented in [Pod restart issue](https://github.com/istio/istio/issues/4822)
+
+
+There are a number of ways to either avoid or resolve this scenario. This section 
+provides a high level overview of these options.   
+
+* Update the dns entries 
+* Use a load balancer service type 
+* Expose the Istio services via a gateway
+
+### Update the dns entries
+
+Upon any failure or pod restart kube-dns on the remote clusters can be
+updated with the correct endpoint mappings for the Istio services.  There 
+are a number of ways this can be done the most obvious is to rerun the Helm 
+install after the Istio services on the control plane cluster have restarted. 
```

Comment:
How about

s/rerun the Helm install/rerun the Helm install in remote cluster

Reply:
sure will take care of it